atat_provisioning_wizard_api.yaml

```diff
@@ -298,7 +357,13 @@ paths:
           application/json:
             schema:
               $ref: '#/components/schemas/ApplicationStep'
-  '/portfolioDrafts/{portfolioDraftId}/submit':
+      x-amazon-apigateway-integration:
+          httpMethod: "POST"
+          uri: 
+            Fn::Sub: "arn:${AWS::Partition}:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${GetPortfolioDraftsFunction.Arn}/invocations"
```

Comment:
This and all unimplemented functions need to use a new function that returns a "Not Implemented" response (either 400 or 404. see if @jeffsegal-ccpo has a recommendation). That function should probably go ahead and use our ErrorResponse and can be reused.

Make sure when doing the find-and-replace that you don't accidentally replace the actual GetPortfolioDraftsFunction with the new NotImplementedFunction

Reply:
Done.